src/BreathOfTheWild/Mods/MasterModeOptions/patch_patches.asm

```diff
@@ -0,0 +1,18 @@
+[BotwV208]

```

Comment:
Could you rename the file to `patch_MasterModeOptions.asm` or `patch_MasterModeTweaks.asm`.
Also, please change the name of the patch to something more specific like `[BotW_MasterModeOptions_V208]` or `[BotW_MasterModeTweaks_V208]`.